fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/FirstOrDefaultValue.java

```diff
@@ -110,17 +110,6 @@ public <M extends Message> Object eval(@Nonnull final FDBRecordStoreBase<M> stor
         return childrenObjects.get(0);
     }
 
-    @Override
```

Comment:
This was a bug where this method `equalsWithoutChildren()` actually looked at its children thus doubling the effort.